tapeagents/finetune/rl/__init__.py

```diff
@@ -128,6 +107,7 @@ def rl_step(model, batch, config: RLConfig) -> tuple[torch.Tensor, dict[str, flo
     log_ratio_new_old = new_log_probs - old_logprobs
     ratio_new_old = torch.exp(log_ratio_new_old)
     weights = advantages if config.use_advantages else rewards
+    weights = torch.clamp(weights, min=0) if config.relu_weights else weights
```

Comment:
where does this trick come from? can you put a paper reference? where do we use it?